test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ApiBehaviorApplicationModelProviderTest.cs

```diff
@@ -556,6 +558,57 @@ public void PreservesBindingSourceInference_ForFromQueryParameter_WithCustomName
         {
             // Arrange
             var modelMetadataProvider = TestModelMetadataProvider.CreateDefaultProvider();
+=======
```

Comment:
@kishanAnem please resolve these conflicts. That is, accept `dev` additions and your additions to reduce the visible changes to what you intended.

Reply:
Sure, working on it. This does not actually commit